knowledge-base/customize-appointment-display-dot-net-maui-scheduler.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Customizing Appointments in Telerik UI for .NET MAUI Scheduler
+description: Learn how to customize the appointment display in the Telerik UI for .NET MAUI Scheduler to show a dot next to the appointment text instead of a background color.
+type: how-to
+page_title: How to Customize Appointment in .NET MAUI Scheduler
+slug: customize-appointment-display-dot-net-maui-scheduler
+tags: calendar, .net maui, scheduler, appointment, template, customization
+res_type: kb
+ticketid: 1654745
+---
+
+## Environment
+
+| Version | Product | Author | 
+| --- | --- | ---- | 
+| 7.0.0 | Telerik UI for .NET MAUI Scheduler | [Dobrinka Yordanova](https://www.telerik.com/blogs/author/dobrinka-yordanova)| 
+
+## Description
+
+In migrating from Telerik UI for Xamarin to Telerik UI for .NET MAUI, I need to display a small colored dot next to the appointment in the month view in the [Scheduler]({%slug scheduler-overview%}), rather than using a background color.
```

Comment:
```suggestion
I am migrating my scheduler functionality from Telerik UI for Xamarin to Telerik UI for .NET MAUI. I need to display a small colored dot next to the appointment in the month view in the [Scheduler]({%slug scheduler-overview%}), rather than using a background color.
```